client/deis.py

```diff
@@ -316,13 +316,14 @@ def readable_datetime(datetime_str):
             return 'Just now'
         else:
             return "{}{}ago".format(hour_str, min_str)
+
     # if it happened yesterday, say "yesterday at 3:23 pm"
-    yesterday = now + relativedelta.relativedelta(days=-1)
-    if delta.days <= 2 and dt.day == yesterday.day:
```

Comment:
I think this logig's a bit cleaner and accomplishes the same thing as line 322. Am I missing something?